RELEASE_PROCESS.md

```diff
@@ -0,0 +1,137 @@
+# Podman Releases
+
+## Definitions
+
+## Expectations
+
+* You have push access to the [upstream podman repository][https://github.com/containers/podman.git]
+* You understand all basic `git` operations and concepts, like creating commits,
+  local vs. remote branches, rebasing, and conflict resolution.
+* You have access to your public and private GPG keys.
+* You have reliable internet access (i.e. not the public WiFi link at McDonalds)
+* Other podman maintainers are online/available for assistance if needed.
+* For a major release, you have 4-8 hours of time available, most of which will
+  be dedicatd to writing release notes.
+* For a minor or micro release, you have 2-4 hours of time available (minimum depends
+  largely on the speed/reliability of automated testing)
+
+# Release Process
+
+## Overview
+
+## Micro version (`*.*.Z`)
```

Comment:
[semver](https://semver.org/) refers to that as the "PATH" version.

Reply:
yes, I agree "patch" is better.  I'll fix it.